controllers/k8ssandra/k8ssandracluster_controller.go

```diff
@@ -86,6 +91,13 @@ func (r *K8ssandraClusterReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	patch := client.MergeFromWithOptions(kc.DeepCopy())
 	result, err := r.reconcile(ctx, kc, logger)
 	if kc.GetDeletionTimestamp() == nil {
+		if err != nil {
+			kc.Status.Error = err.Error()
+			r.Recorder.Event(kc, core.EventTypeWarning, "Reconcile Error", err.Error())
+		} else {
+			kc.Status.Error = "None"
+			r.Recorder.Event(kc, core.EventTypeNormal, "Updated", fmt.Sprintf("Updated K8ssandraCluster %s/%s", kc.Namespace, kc.Name))
```

Comment:
This feels odd event. We don't really update K8ssandraCluster, other than the Status. But I don't see that as an event that we should notify the user about.

Reply:
I was wondering if that was a good idea. I'll remove it.